gtd/gtd-plans.js

```diff
@@ -0,0 +1,142 @@
+// GTD plans fetch
+
+function extractProductDetails(text) {
+  const internetPrefix = "INTERNET FIBRA ÓPTICA";
+  const phonePrefix = "TELEFONÍA Plan Minutos Libre";
+  if (text.startsWith(internetPrefix)) {
+    // example: INTERNET FIBRA ÓPTICA 200 Mbps  60 Mbps Incluye Wifi HI Speed y 2 Smart Connect
+    const parts = text
+      .replace(internetPrefix, "")
+      .trim()
+      .split(" ");
+    return {
+      productType: "internet",
```

Comment:
Product types may be an object defined outside, so as to avoid hardcoding these strings